.github/pull_request_template.md

```diff
@@ -1,58 +1,38 @@
 
-<!--
-Thank you for opening a pull request!  Here are some tips on creating
-a well formatted contribution.
-
-Please give your pull request a title like "[component]: [short description]"
-
-This is the format for commit messages:
 
-"""
-[component]: [short description]
+<!--
+  - Please give your pull request a title like
 
-[A longer multiline description]
+      [component]: [short description]
 
-Fixes: [ticket URL on tracker.ceph.com, create one if necessary]
-Signed-off-by: [Your Name] <[your email]>
-"""
+  - Please use this format for each git commit message:
 
-The Signed-off-by line is important, and it is your certification that
-your contributions satisfy the Developers Certificate or Origin.  For
-more detail, see SubmittingPatches.rst.
+      [component]: [short description]
 
-The component is the short name of a major daemon or subsystem,
-something like "mon", "osd", "mds", "rbd, "rgw", etc. For ceph-mgr modules,
-give the component as "mgr/<module name>" rather than a path into pybind.
+      [A longer multiline description]
 
-For more examples, simply use "git log" and look at some historical commits.
+      Fixes: [ticket URL on tracker.ceph.com, create one if necessary]
+      Signed-off-by: [Your Name] <[your email]>
```

Comment:
Same thing. If we really want to enforce this I'd go for automation and extending the current sign-off check (otherwise it's subject to unintentional omission by reviewers). We could even validate the component from a predefined list of names.

Reply:
Are you suggesting we remove this from the pull request template?  (I'm all for it!  The less clutter the better.)